__test__/Home.test.jsx

```diff
@@ -1,6 +1,6 @@
 import '@testing-library/jest-dom';
 import { render, screen } from '@testing-library/react';
-import Home from '../app/page';
+import Home from '../src/app/page';
```

Comment:
🟢 We can use absolute path imports for cleaner references and easier refactoring. I can go ahead and update Jest.config.ts and tsconfig.json to support absolute paths for our new project folder structure .